pkg/mapper/configmap/configmap_test.go

```diff
@@ -266,7 +266,7 @@ func TestParseMap(t *testing.T) {
 	}
 	accounts := []string{}
 
-	u, r, a, err := ParseMap(m1)
+	u, r, _, a, err := ParseMap(m1)
```

Comment:
Please add a unit test to validate configuration file parsing for the newly-added config elements.